src/main/resources/com/github/checkstyle/regression/configuration/base_config.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<module name="Checker">
+  <property name="charset" value="UTF-8"/>
+  <!-- do not change severity to 'error', as that will hide errors caused by exceptions -->
+  <property name="severity" value="warning"/>
+  <!-- haltOnException is required for exception fixes and reporting of all exceptions -->
+  <property name="haltOnException" value="false"/>
+  <!-- BeforeExecutionFileFilters is required for sources of java9 -->
+  <module name="BeforeExecutionExclusionFileFilter">
+    <property name="fileNamePattern" value="module\-info\.java$"/>
+  </module>
+
+  <!-- Contents below are generated by regression-tool. -->
+  <module name="TreeWalker"/></module>
```

Comment:
I don't like this either, but there would be a strange indention problem if I put `</module>` to next line.

i.e.
```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.3//EN" "http://checkstyle.sourceforge.net/dtds/configuration_1_3.dtd">
<module name="Checker">
  ...
  <!-- Contents below are generated by regression-tool. -->
  <module name="TreeWalker"/>
<module name="FileLengthCheck"/>
</module>
```

You could have a try to put that to next line, I assume `testGenerateConfigTextWithEmptyModuleInfos` and `testGenerateConfigTextWithCheckerParentModule` would fail due to that indention problem